src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector128.cs

```diff
@@ -1172,7 +1172,7 @@ public static unsafe Vector128<short> CreateScalar(short value)
             Vector128<short> SoftwareFallback(short x)
```

Comment:
Can you make it `static` to enforce this? Should be using latest C#

Reply:
I believe this would require us to explicitly set langVer to 8. Not sure if that is something the entire repo wants to do yet.

Reply:
> I believe this would require us to explicitly set langVer to 8. Not sure if that is something the entire repo wants to do yet.

Do we even have a compiler in coreclr/corefx that understand langver 8 yet?  Last I checked we didn't, but maybe that changed recently?

Reply:
`System.Private.CoreLib.csproj` uses `<LangVersion>latest</LangVersion>`

https://github.com/dotnet/coreclr/blob/930abba4060fb528db2bb9835a1bc5a6e684bfec/src/System.Private.CoreLib/System.Private.CoreLib.csproj#L38

Don't know what version of the compiler it pulls in though; I was assuming; perhaps incorrectly, that since is part of the [Preview 1 of .NET Core 3.0 and Preview 1 of Visual Studio 2019](https://blogs.msdn.microsoft.com/dotnet/2018/12/05/take-c-8-0-for-a-spin/) mix that coreclr/corefx would have been dogfooding it?

Reply:
> Do we even have a compiler in coreclr/corefx that understand langver 8 yet? Last I checked we didn't, but maybe that changed recently?

Looks like no. We are still using 2.9.0.beta8-63208-01.

Reply:
> that coreclr/corefx would have been dogfooding it?

It looks like in the interim we actually have updated to using a preview compiler for C# 8, at least in corefx.  Nice.

Reply:
Having to rename params to something else for local functions; rather than shadowing them, is the thing that bugs me the most; which static local methods solves.

Reply:
@stephentoub, where are you seeing 2.11? At least locally (looking at the binlogs, etc) we seem to be using 2.9.0-beta8...

Reply:
In corefx, looking at the output from a dotnet msbuild /v:d, getting the path from there of the csc.dll being used, and running dotnet csc.dll to get the version info.  I also tried using C# 8 features in Preview 1, and they worked fine.

Reply:
`dotnet msbuild` wouldn't be the SDK used by `build.cmd` however? It should be the `dotnet` from whatever SDK is on your path, which is separate.

Reply:
That's possible.  But that also means we've set ourselves up for failure, as a developer working on a library might be using a different compiler than official builds.

Reply:
Should be at least the compiler that comes with 3.0 preview 2 sdk as its since its post preview 2 code?  https://blogs.msdn.microsoft.com/dotnet/2019/01/29/announcing-net-core-3-preview-2/ which is C#8.

Otherwise there's a mismatch going on?

Reply:
I don't believe CoreFX is pulling down the preview 3 SDK. From the looks of it, it is pulling down (and using) 2.1.401

Reply:
Latest doesn't trigger it; needs to be 8.0 explicitly. Seeing if it works https://github.com/dotnet/coreclr/pull/22272